package.json

```diff
@@ -79,9 +79,9 @@
     "@types/uuid": "^9.0.7",
     "better-react-mathjax": "^2.0.3",
     "buffer": "^6.0.3",
-    "change-case": "^4.1.2",
+    "change-case": "^5.4.4",
```

Comment:
@kimon-satan you need to branch this PR off `main` (not the PR branch) if you're intending to merge into `main`

Reply:
whoops. I meant to merge into the PR branch . let me redoit